wcfsetup/install/files/lib/system/acp/dashboard/box/UsersAwaitingApprovalAcpDashboardBox.class.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace wcf\system\acp\dashboard\box;
+
+use wcf\data\user\User;
+use wcf\data\user\UserProfileList;
+use wcf\system\WCF;
+
+/**
+ * ACP dashboard box listing users waiting for approval.
+ *
+ * @author      Marcel Werk
+ * @copyright   2001-2023 WoltLab GmbH
+ * @license     GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @since       6.1
+ */
+final class UsersAwaitingApprovalAcpDashboardBox extends AbstractAcpDashboardBox
+{
+    private UserProfileList $userList;
+    private int $usersAwaitingApproval;
+
+    #[\Override]
+    public function isAccessible(): bool
+    {
+        return WCF::getSession()->getPermission('admin.user.canSearchUser')
+            && REGISTER_ACTIVATION_METHOD & User::REGISTER_ACTIVATION_ADMIN;
```

Comment:
```suggestion
            && (REGISTER_ACTIVATION_METHOD & User::REGISTER_ACTIVATION_ADMIN);
```
Probably just copy-pasted, but we should make precedence clear while we're at it.